File: qlty-cli/tests/lang/java/basic.in/lcom/lcom_2.java
// lcom=2

class KlassA {
  String aaa;
  String bbb;

  public String getBbb() {
    return this.aaa;
  }

  public String getAaa() {
    return this.aaa;
  }

  String foo() {
    return this.getAaa();
  }

  String bar() {
    return this.getBbb();
  }
}

class KlassB {
  String baz;

  public String getBar() {
    return this.baz;
  }

  String foo() {
    return this.getBar();
  }

  String bar() {
    return this.getBar();
  }
}

File: qlty-cli/tests/lang/csharp/basic.in/lcom/lcom_2.cs
// lcom=2

class KlassA
{
    private string aaa;
    private string bbb;

    public string GetBbb()
    {
        return this.bbb;
    }

    public string GetAaa()
    {
        return this.aaa;
    }

    private string Foo()
    {
        return this.GetAaa();
    }

    private string Bar()
    {
        return this.GetBbb();
    }
}

class KlassB
{
    private string baz;

    public string GetBar()
    {
        return this.baz;
    }

    private string Foo()
    {
        return this.GetBar();
    }

    private string Bar()
    {
        return this.GetBar();
    }
}